### ISSUE TITLE
🐛 No node found for selector [aria-label='Scale 3 x']

### ISSUE THREAD

**Issue by jsjoeio** [excalidraw/gatsby-embedder-excalidraw#22]
I'm not sure if I should file here or as an issue with `gatsby-plugin-mdx` but after a month of not touching my site, I can't get the development bundle to build correctly and it looks like it's unhappy with the Excalidraw links.

```shell
yarn run v1.22.4
$ gatsby develop
success open and validate gatsby-configs - 0.092s
success load plugins - 4.334s
success onPreInit - 0.018s
success initialize cache - 0.022s
success copy gatsby files - 0.075s
success onPreBootstrap - 0.013s
success createSchemaCustomization - 0.005s

 ERROR #11321  PLUGIN

"gatsby-plugin-mdx" threw an error while running the onCreateNode lifecycle:

The following error appeared while processing 'https://excalidraw.com/#json=5086916491870208,sLUFfW5oMgrO3ApQwe4YQw':

No node found for selector: [aria-label='Scale 3 x']

  31 |   const createAndProcessNode = path => {
  32 |     const fileNodePromise = createFileNode(path, createNodeId, pluginOptions).then(fileNode => {
> 33 |       createNode(fileNode);
     |       ^
  34 |       return null;
  35 |     });
  36 |     return fileNodePromise;

File: node_modules/gatsby-source-filesystem/gatsby-node.js:33:7
```

## To Reproduce

This is on [my personal site](https://github.com/jsjoeio/joeprevite.com) on the `bug-reproduce`.

1. `git clone --single-branch --branch bug-reproduce https://github.com/jsjoeio/joeprevite.com.git`
2. `yarn install`
3. `yarn dev`

Observe output

### Expected

Success - everywhere!

### Actual

A bunch of errors and an unhappy development bundle

**Comment by jsjoeio** [comment]
When I comment out all the Excalidraw links, everything works as expected. No errors.

**Comment by trevorblades** [comment]
Indeed, this error was happening because the element with the former selector `[aria-label='Scale 3 x']` was replaced on the Excalidraw website. I just removed this line and published a new version of the package, so it should work now.

I'm not worried about the lack of scaling x3 since we're talking about infinitely scalable SVGs anyway 🤷 

`npm install gatsby-embedder-excalidraw@latest` should do the trick! 🤞
